bundles/action/org.openhab.action.pushover/src/main/java/org/openhab/action/pushover/internal/Pushover.java

```diff
@@ -226,13 +226,17 @@ public static boolean pushover(@ParamDoc(name = "apiKey", text = "Your applicati
 
             if (!StringUtils.isEmpty(apiKey)) {
                 addEncodedParameter(data, MESSAGE_KEY_API_KEY, apiKey);
+            } else if (StringUtils.isEmpty(apiKey) && !StringUtils.isEmpty(defaultApiKey)) {
+                addEncodedParameter(data, MESSAGE_KEY_API_KEY, defaultApiKey);
             } else {
                 logger.error("Application API token not specified.");
                 return false;
             }
 
             if (!StringUtils.isEmpty(user)) {
                 addEncodedParameter(data, MESSAGE_KEY_USER, user);
+            } else if (StringUtils.isEmpty(user) && !StringUtils.isEmpty(defaultUser)) {
```

Comment:
The part `StringUtils.isEmpty(user) && ` is unneeded and would read better if it were removed.